docs/sdk/serverless-signing/get-rpc-url.md

```diff
@@ -0,0 +1,62 @@
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+# Get Chain RPC URLs Within an Action
+
+## Overview
+
+You can use the getRpcUrl function within a Lit Action to call make an RPC call to a given blockchain. This can be useful for sending transactions, calling contract methods, pulling block data, and other related use cases.
```

Comment:
```suggestion
You can use the `getRpcUrl` function within a Lit Action to call make an RPC call to a given blockchain. This can be useful for sending transactions, calling contract methods, pulling block data, and other related use cases.
```